script/refresh_sitemap

```diff
@@ -128,7 +136,10 @@ ROBOTS_HEADER = <<~"TXT"
   Disallow: */lookup*user_locale=*
   Disallow: /*?*q= # block urls having the q parameter
   Disallow: /name/show_name
```

Comment:
This was blocked 6 months ago for performance reasons. See c6ee56e
In the long run, it should be unblocked. But I favor keeping it this way for the moment until we block all unregistered users.
What do others think?